version.cake

```diff
@@ -36,4 +38,78 @@ void UpdateSDKVersion(string path, string patternPrefix, string newVersion)
     ReplaceRegexInFiles(path, patternString, newString);
 }
 
+Task("UpdateCgManifestSHA").IsDependeeOf("StartNewVersion").Does(()=>
+{
+    try
+    {
+        var reposToUpdate = new List<string>(){ 
+            "https://github.com/microsoft/appcenter-sdk-android.git",
+            "https://github.com/microsoft/appcenter-sdk-apple.git",
+            "https://github.com/microsoft/appcenter-sdk-dotnet.git"
```

Comment:
Move it to costs to correlate to specified versions

Reply:
Fixed